### PR TITLE
feat: add text to string types

### PR DESCRIPTION
**Pull request by rawestmoreland**
Add type `text` to be able to see `long text`.

### CODE REVIEW

**Comment by MaksZhukov** [comment]
> @MaksZhukov Do you use tabs?
yes

**Comment by MaksZhukov** [comment]
Also add here 
image
and here 
image

File: admin/src/components/attributeTypeComponents/StringInput/StringInput.tsx [hunk 1]
```diff
@@ -31,7 +31,8 @@ const StringInput = ({
 						onChange={onChangeCheck(attributeKey)}
 						checked={checked}>
 						{`${attributeKey} (Field type: ${
-							attribute.type === AttributeType.String
+							attribute.type === AttributeType.String ||
+              						attribute.typp == AttributeType.Text
```
Comment (MaksZhukov) [inline admin/src/components/attributeTypeComponents/StringInput/StringInput.tsx:35]: change typp to type

Comment (MaksZhukov) [reply admin/src/components/attributeTypeComponents/StringInput/StringInput.tsx:35]: @rawestmoreland

**Comment by MaksZhukov** [comment]
And last thing add here 
image

File: admin/src/components/attributeTypeComponents/StringInput/StringInput.tsx (hunk 1 above)
Comment (rawestmoreland) [reply admin/src/components/attributeTypeComponents/StringInput/StringInput.tsx:35]: 🤦🏼‍♂️

**Review by MaksZhukov — APPROVED** [review]

**Comment by MaksZhukov** [comment]
Great! Thanks!

**Pull request merged**